src/Controls/GlobeControls.js

```diff
@@ -178,9 +179,15 @@ class GlobeControls extends THREE.EventDispatcher {
         // Set to false to disable this control
         this.enabled = true;
 
-        // This option actually enables dollying in and out; left as "zoom" for
+        // These options actually enables dollying in and out; left as "zoom" for
         // backwards compatibility
-        this.zoomSpeed = options.zoomSpeed || 2.0;
+        if (options.zoomSpeed) {
+            console.warn('Controls zoomSpeed parameter is deprecated. Use zoomInFactor and zoomOutFactor instead.');
+            options.zoomInFactor = options.zoomInFactor || options.zoomSpeed;
+            options.zoomOutFactor = options.zoomOutFactor || (1 / options.zoomSpeed);
+        }
+        this.zoomInFactor = options.zoomInFactor || 2.0;
```

Comment:
I ask whether it is necessary to have two parameters (in and out) when one is the inverse of the other.
WDYT?

Reply:
I changed `GlobeControl` as well as `PlanarControls` so that user passes a single `zoomFactor` parameter (which is equal to the previous `zoomInFactor` parameter). The `zoomOutFactor` is now computed as the inverse of `zoomFactor`.